Sming/Libraries/DIAL/sample/app/application.cpp

```diff
@@ -7,7 +7,9 @@
 #define WIFI_PWD "PleaseEnterPass"
 #endif
 
-static Dial::Client client;
+namespace
```

Comment:
@mikee47 I will remove the anonymous namespace. If we apply this change it should be applied to all samples for consistency. Which can be part of a separate PR.